src/Configuration/test/ConfigServer.ITest/ConfigServerConfigurationExtensionsIntegrationTest.cs

```diff
@@ -433,7 +439,9 @@ public async void SpringCloudConfigServer_WithHealthEnabled_ReturnsHealth()
             using var server = new TestServer(builder);
             var client = server.CreateClient();
             var result = await client.GetStringAsync("http://localhost/Home/Health");
-            Assert.Equal("UP,https://github.com/spring-cloud-samples/config-repo/foo-development.properties,https://github.com/spring-cloud-samples/config-repo/foo.properties,https://github.com/spring-cloud-samples/config-repo/application.yml,", result);
+
+            // after switching to new config server image, this value now ends with " (document #0),"
```

Comment:
This new text is indeed a feature and not a bug - the # increments with stacked config files